web/src/tippyjs.js

```diff
@@ -470,7 +470,7 @@ export function initialize() {
     });
 
     delegate("body", {
-        target: ".revoke-invite, resend-invite",
+        target: ".edit-invite, .revoke-invite, .resend-invite",
```

Comment:
The "resend-invite" -> ".resend-invite" change should be done in the commit which adds the tooltip. We try to include unrelated changes in the commit to maintain a clean history.